<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppView</class>
 <widget class="QMainWindow" name="AppView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::TextElideMode::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabReserves">
       <attribute name="title">
        <string>Reservas</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutReserves">
        <item>
         <widget class="QWidget" name="widContentReserves" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabRooms">
       <attribute name="title">
        <string>Quartos</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutRooms">
        <item>
         <widget class="QWidget" name="widContentRooms" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabGuests">
       <attribute name="title">
        <string>HÃ³spedes</string>
       </attribute>
       <layout class="QVBoxLayout" name="layoutGuests">
        <item>
         <widget class="QWidget" name="widContentGuests" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
